src/collection/utils/populate.ts

```
@@ -0,0 +1,123 @@
+import { Sort as MongoSort } from "mongodb";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)